modules/api/src/main/java/org/apache/ignite/security/AuthenticationType.java

```diff
@@ -17,24 +17,7 @@
 
 package org.apache.ignite.security;
 
-import java.util.Arrays;
-import org.apache.ignite.security.exception.UnsupportedAuthenticationTypeException;
-
 /** Authentication types. */
 public enum AuthenticationType {
-    BASIC;
-
-    /**
-     * Parses {@link AuthenticationType} from the given string.
-     *
-     * @param type String representation of the type.
-     * @return parsed {@link AuthenticationType}.
-     * @throws UnsupportedAuthenticationTypeException in case of unknown type.
-     */
-    public static AuthenticationType parse(String type) {
-        return Arrays.stream(values())
-                .filter(it -> type.equalsIgnoreCase(it.name()))
-                .findFirst()
-                .orElseThrow(() -> new UnsupportedAuthenticationTypeException("Unknown authentication type: " + type));
-    }
+    PASSWORD;
```

Comment:
See my comment in the [JIRA ticket](https://issues.apache.org/jira/browse/IGNITE-21064) please - I'm not sure what are we trying to improve here.